### PR TITLE
fix: encode revocation tag in tails upload URL (issue 1580)

### PR DESCRIPTION
**Pull request by sonivijayk**
## Fix Summary
- URL-encode revocation registry id when building tails upload/public URLs
- Add test coverage for revocation tags containing whitespace

## Fix Testing
- Ran python -m pytest acapy_agent/tails/tests/test_indy.py

### CODE REVIEW

File: acapy_agent/tails/tests/test_indy.py [hunk 1]
```diff
@@ -40,10 +42,14 @@ async def test_upload(self):
             assert ok
 
             # already contains / from config, no need to add it
```
[the remark this answers is not in the document]
Comment (sonivijayk) [reply acapy_agent/tails/tests/test_indy.py]: Comment is not needed so removed

File: acapy_agent/tails/tests/test_indy.py [hunk 2]
```diff
@@ -79,10 +85,44 @@ async def test_upload_indy_vdr(self):
...
```
[the remark this answers is not in the document]
Comment (sonivijayk) [reply acapy_agent/tails/tests/test_indy.py]: Comment is not needed so removed

**Comment by jamshale** [comment]
Thanks for your contribution. I think this is good and will approve. 

You do need to fix the DCO check. There's some instructions for fixing this in the action details. https://github.com/openwallet-foundation/acapy/pull/3996/checks?check_run_id=58834216130

**Comment by sonivijayk** [comment]
@jamshale Thank you. I have fixed DCO sign off.

**Review by jamshale — APPROVED** [review]
